sympy/geometry/ellipse.py

```diff
@@ -1320,6 +1315,15 @@ class Circle(Ellipse):
         When trying to construct circle from three collinear points.
         When trying to construct circle from incorrect parameters.
 
+    If the equation of a circle, whose circle object is needed, is
```

Comment:
This can be moved into the Examples section

Reply:
@smichr I think `# a circle object returned from given equation and optional 'x' and 'y' parameters` statement explains the example (in the `Examples` section) quite satisfactorily. But `If the equation of a circle, whose circle object is needed ... . . ` needs to be in the documentation since it explains another parameter.